explainer.md

```diff
@@ -72,6 +72,113 @@ There are still many open questions, but a few big ones:
 - Will existing protocols ([example](https://github.com/openid/OpenID4VP/issues/125)) adopt this API?
 - Will regulation ([example](https://digital-strategy.ec.europa.eu/en/library/european-digital-identity-architecture-and-reference-framework-outline)) adopt this API?
 
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+> 01.  What information does this feature expose,
+>      and for what purposes?
+
+The Digital Credential API exposes a one-time user-mediated end-to-end encrypted communication channel from websites, to digital wallet applications, and back to websites. It is designed to be a better option than established lower-level communication channels like custom schemes, QR codes, and server-to-server network communication. How exactly this channel is used is up to the wallet applications and host operating system, but we are designing it to be suitable for conveying presentations of digital credentials such as claims in mobile driver’s licenses.
+
+> 02.  Do features in your specification expose the minimum amount of information
+>      necessary to implement the intended functionality?
+
+A primary use case of the digital credentials API is for selective disclosure of identity properties such as a cryptographic attestation that the user holds a California driver’s license of an adult. The use of selective disclosure, however, is a decision for the verifier website, wallet and credential issuer based on the use case. There are legitimate scenarios, such as creating or recovering an account on a government website, where uniquely identifiable and potentially non-resettable PII is potentially exposed.
+
+The API is designed to expect the use of response encryption so that this PII is exposed only to the requesting server and not any code running in the web page or browser. It is an [open question](https://github.com/WICG/digital-identities/issues/109) whether this response encryption is something we can reasonably enforce at this layer or not.
```

Comment:
```
whether this response encryption is something we can reasonably enforce at this layer or not.
```

It doesn't seem to me like it is an open question whether this layer can strictly enforce response encryption: it can't.

There is nothing that the browser layer can do (outside of curated allowlists) to circumvent a malicious (or not) wallet from returning the response in plain text.

Reply:
I expanded on this a bit more here: https://github.com/WICG/digital-identities/issues/109#issuecomment-2096424454

Reply:
```suggestion
The API is designed to expect the use of response encryption so that this PII is exposed only to the requesting server and not to any code running in the web page or browser. It is an [open question](https://github.com/WICG/digital-identities/issues/109) whether such response encryption is something we can reasonably enforce at this layer or not.
```

Reply:
> It doesn't seem to me like it is an open question whether this layer can strictly enforce response encryption: it can't.

Changed "enforce" to "require". As discussed in the issue, our spec could require it even if that can't be technically enforced.